htdocs/core/class/commonobject.class.php

```diff
@@ -4200,7 +4201,7 @@ public function printOriginLinesList($restrictlist = '')
 	 */
 	public function printOriginLine($line, $var, $restrictlist = '', $defaulttpldir = '/core/tpl')
 	{
-		global $langs, $conf;
+		global $langs, $conf, $selectedLines;
```

Comment:
To try to keep use of global $var only for very global need, can you try by adding
$selectedLines as parameter of printOriginLineList that propagate then to printOriginLine